src/Modules/Backend/Domain/ModuleSettings/ModuleSettingsType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\ModuleSettings;
+
+use ForkCMS\Core\Domain\Form\SwitchType;
+use ForkCMS\Core\Domain\Form\TogglePasswordInputType;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use ForkCMS\Modules\Internationalisation\Domain\Translation\TranslationKey;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Expression;
+
+final class ModuleSettingsType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add(
+                '2fa_enabled',
+                CheckboxType::class,
+                [
+                    'required' => false,
+                    'label' => TranslationKey::label('EnableTwoFactorAuthentication'),
+                    'label_attr' => ['class' => 'checkbox-switch'],
```

Comment:
There is a switch type in fork already that you can use

Reply:
fixed f92c386e984adc1cf4173106876f7a1d10453afe